index.html

```diff
@@ -102,112 +102,10 @@ <h2>LET'S CONNECT</h2>
-            <div class="media-img img-2">
-              <img src="assets/images/SnapshootPortfolio3.png" alt="Project Image" />
-            </div>
-            <div class="media-body">
-              <h4> Multi-Post <span> Stories</span>
-              </h4>
-              <div class="role">
-                <span>CANOPY</span>
-                <i></i>
-                <span>Backend Dev</span>
-                <i></i>
-                <span>2015</span>
-              </div>
-              <div class="description">
-                <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
-              </div>
-              <ul class="skills">
-                <li>html</li>
-                <li>css</li>
-                <li>javasript</li>
-              </ul>
-              <div class="read-more">
-                <a href="#"> See Project </a>
-              </div>
-            </div>
-          </article>
-          <article class="media">
-            <div class="media-img img-3">
-              <img src="assets/images/SnapshootPortfolio2.png" alt="Project Image" />
-            </div>
-            <div class="media-body">
-              <h4>Tonic</h4>
-              <div class="role">
-                <span>CANOPY</span>
-                <i></i>
-                <span>Backend Dev</span>
-                <i></i>
-                <span>2015</span>
-              </div>
-              <div class="description">
-                <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
-              </div>
-              <ul class="skills">
-                <li>html</li>
-                <li>css</li>
-                <li>javasript</li>
-              </ul>
-              <div class="read-more">
-                <a href="#"> See Project </a>
-              </div>
-            </div>
-          </article>
-          <article class="media">
-            <div class="media-img img-4">
-              <img src="assets/images/SnapshootPortfolio.png" alt="Project Image" />
-            </div>
-            <div class="media-body">
-              <h4>Multi-Post <span>Stories</span>
-              </h4>
-              <div class="role">
-                <span>CANOPY</span>
-                <i></i>
-                <span>Backend Dev</span>
-                <i></i>
-                <span>2015</span>
-              </div>
-              <div class="description">
-                <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
-              </div>
-              <ul class="skills">
-                <li>html</li>
-                <li>css</li>
-                <li>javasript</li>
-              </ul>
-              <div class="read-more">
-                <a href="#"> See Project </a>
-              </div>
-            </div>
-          </article>
+         
+        </section>
+        <section class="portfolio-popup">
```

Comment:
- [ ] Good job adding your popup. It looks responsive 😄 . What is left is to add the appropriate padding around the modal on both the desktop and mobile view to match the figma design 👇🏽. Yours seems to be touching the top of the page and the background is not exactly as its on figma.

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/216433931-0bdaf71e-49ac-44be-b817-862580af30b0.png)
![image](https://user-images.githubusercontent.com/63473584/216434125-6010b88e-6c2c-4254-b5c0-708863ff910d.png)

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/216434235-8a32a141-bfa8-4464-a6de-b9d5e5adb413.png)

![image](https://user-images.githubusercontent.com/63473584/216434433-b2191e5e-f253-441e-b266-385d8bc68fb0.png)